### ISSUE TITLE
The media key does not work very well and suddenly it does not work at all

### ISSUE THREAD

**Issue by TOP-47** [th-ch/youtube-music#364]
I have a razer cynosa v2 which is 1 month old and the media keys (bug) and sometimes it works perfectly which is a bit annoying knowing that I listen to 100% of my music thanks to your application

sorry for my bad english i'm french and i use google traduction

**Comment by thijsi123** [comment]
Just asking,
Is youtube/streaming service on chrome, brave, etc... or any other app that could use the media keys open in the background?
In my case, I always have brave open with youtube or animixplay that prioritize the media keys for some reason so I have to close those websites for the media keys to work. That's all I can help with.

**Comment by TOP-47** [comment]
yes sometimes it is open but it still works sometimes nothing is open but the media keys do not work

**Comment by DoomTroopa84** [comment]
I have a Steelseries Apex 7 and the play pause doesn't work however the same button works to skip/go back tracks
